corehq/apps/es/es_query.py

```diff
@@ -353,12 +353,20 @@ def pprint(self):
         """pretty prints the JSON query that will be sent to elasticsearch."""
         print self.dumps(pretty=True)
 
-    def sort(self, field, desc=False):
+    def sort(self, field, desc=False, reset_sort=True):
```

Comment:
there're a bunch of tests for these i think and could be easy to make one for this too